.github/workflows/release.yml

```diff
@@ -60,13 +60,18 @@ jobs:
 
           if [ "${{ matrix.target }}" == "win-x64" ]; then
             7z a -tzip "${release_name}.zip" "./${release_name}/*"
+
+            # create .nupkg for NuGet
+            dotnet pack ThunderstoreCLI/ThunderstoreCLI.csproj -c Release -o "nuget/" -p:SelfContained=false -p:PublishTrimmed=false -p:PublishSingleFile=false -p:StartupObject="" -p:RuntimeIdentifier="" -p:PublishReadyToRun=false -p:PackAsTool=true
```

Comment:
Why is this inside the same windows platform if clause, aren't nuget packages cross platform??

Reply:
Presumably this is to ensure it only runs once

Reply:
That's correct, since it only needs to be built once for all platforms I only build it on Windows instead of building it on each platform

Reply:
How about making it it's own job/matrix option? Seems a bit misleading as it is

Reply:
I had thought that making another job with another call to softprops/action-gh-release would overwrite the original, so I tried to avoid that. Seems that's not the case after reading more from it's docs, so I went ahead and moved the build to its own job

Reply:
It's fine the way you did it (at least I'm pretty sure it is), but just for future reference: While generally running the release action multiple times will update the existing release, if it happens to run multiple times at the same exact time it can generate multiple releases instead. I ran into this problem in a repo of mine where I had to set the strategy to only run one matrix at a time to prevent that from happening.